BAZProjectUnitTests/TrendingMoviesViewControllerTests.swift

```diff
@@ -9,27 +9,14 @@ import XCTest
 @testable import BAZProject
 
 final class TrendingMoviesViewControllerTests: XCTestCase {
-    
-    var sut: TrendingMoviesViewController!
-    var viewModel: TrendingMoviesViewModelMock!
-
-    override func setUp() {
-        super.setUp()
-        viewModel = TrendingMoviesViewModelMock()
-        sut = TrendingMoviesViewController(model: viewModel)
-    }
-
-    override func tearDown() {
-        sut = nil
-        super.tearDown()
-    }
 
     func test_NumberOfRowsInSection_ShouldBeThree() {
         //Given
         let movies = [Movie(title: "title1"), Movie(title: "title2"), Movie(title: "title3")]
+        let viewModel = TrendingMoviesViewModelMock(movies: movies, remoteData: TrendingRemoteDataMock())
+        let sut = TrendingMoviesViewController(viewModel: viewModel)
```

Comment:
Parece que tenemos codigo repetitivo, podríamos apoyarnos de un helper method para proteger nuestras pruebas de cualquier cambio , por ejemplo.

```swift
private func makeSUT(movies: [Movie] = []) -> TrendingMoviesViewController {
    let viewModel = TrendingMoviesViewModelMock(movies: movies, remoteData: TrendingRemoteDataMock())
    let sut = TrendingMoviesViewController(viewModel: viewModel)
    return sut
}
```

Y utilizarlo en nuestros metodos de la siguiente manera.

```suggestion
         let sut = makeSUT(movies: movies)
```